src/webgl-surface/drawing/texture/atlas-manager.ts

```diff
@@ -454,14 +454,27 @@ export class AtlasManager {
           );
 
           ctx.font = label.makeCSSFont(fontSize);
+
+          const threeDotsWide = ctx.measureText('...').width;
+          let str = label.text;
+          if (label.maxWidth !== undefined) {
```

Comment:
let's put a space between a block of code and the previous

Also, let's not do an explicit check of undefined and just do

if (!label.maxWidth)

This will also cover the idea that maxWidth of 0 is invalid.